lurk-fibonacci/benches/throughput.rs

```diff
@@ -1,50 +1,36 @@
-use std::{cell::RefCell, rc::Rc, sync::Arc, time::Duration};
-
 use anyhow::anyhow;
+use camino::Utf8PathBuf;
 use criterion::{
     black_box, criterion_group, criterion_main, measurement, BatchSize, BenchmarkGroup,
     BenchmarkId, Criterion, SamplingMode,
 };
-
+use once_cell::sync::Lazy;
 use pasta_curves::pallas;
+use std::{sync::Arc, time::Duration};
+
+use lurk_fibonacci::{fib_expr, fib_frame, fib_limit};
 
 use lurk::{
+    cli::paths::lurk_default_dir,
+    config::lurk_config,
     eval::lang::{Coproc, Lang},
-    field::LurkField,
-    lem::{eval::evaluate, multiframe::MultiFrame, pointers::Ptr, store::Store},
+    lem::{eval::evaluate, multiframe::MultiFrame, store::Store},
     proof::nova::NovaProver,
     proof::Prover,
     public_parameters::{
         instance::{Instance, Kind},
         public_params,
     },
-    state::State,
 };
 
-mod common;
-use common::set_bench_config;
+/// Edit this path to use a config file specific to benchmarking
+/// E.g. `Utf8PathBuf::from("/home/<user>/lurk-rs/lurk-bench.toml");`
+pub(crate) static BENCH_CONFIG_PATH: Lazy<Utf8PathBuf> =
```

Comment:
This is already in `./benches/common`. Can we avoid copying it?

Reply:
No